appyters/GSEA_Appyter/GSEA_Appyter.ipynb

```diff
@@ -36,21 +38,21 @@
     "import math\n",
     "from decimal import Decimal\n",
     "\n",
-    "#loading Enrichr libraries\n",
+    "#loading libraries\n",
     "import requests\n",
     "import urllib.request\n",
+    "from flask import url_for\n",
     "\n",
     "#performing GSEA\n",
     "import gseapy as gp\n",
     "from gseapy.gsea import GSEA\n",
+    "from maayanlab_bioinformatics.dge import limma_voom\n",
+    "from maayanlab_bioinformatics.dge.characteristic_direction import _chdir\n",
```

Comment:
Please do not import private methods, if you need to access chdir directly I recommend using https://github.com/MaayanLab/geode